CHANGELOG.md

```diff
@@ -1129,6936 +1068,13 @@ configuration changes are needed when upgrading.
-  - cors: When `config.credentials = true`, we do not send an ACAO header with
-    value `*`. The ACAO header value will be that of the request's `Origin: `
-    header.
-    [#2451](https://github.com/Kong/kong/pull/2451)
-
-### Fixed
-
-- Upstream connections over TLS now set their Client Hello SNI field. The SNI
-  value is taken from the upstream `Host` header value, and thus also depends
-  on the `preserve_host` setting of your API. Thanks
-  [@konrade](https://github.com/konrade) for the original patch.
-  [#2225](https://github.com/Kong/kong/pull/2225)
-- Correctly match APIs with percent-encoded URIs in their `uris` property.
-  Generally, this change also avoids normalizing (and thus, potentially
-  altering) the request URI when trying to match an API's `uris` value. Instead
-  of relying on the Nginx `$uri` variable, we now use `$request_uri`.
-  [#2377](https://github.com/Kong/kong/pull/2377)
-- Handle a routing edge-case under some conditions with the `uris` matching
-  rule of APIs that would falsely lead Kong into believing no API was matched
-  for what would actually be a valid request.
-  [#2343](https://github.com/Kong/kong/pull/2343)
-- If no API was configured with a `hosts` matching rule, then the
-  `preserve_host` flag would never be honored.
-  [#2344](https://github.com/Kong/kong/pull/2344)
-- The `X-Forwarded-For` header sent to your upstream services by Kong is not
-  set from the Nginx `$proxy_add_x_forwarded_for` variable anymore. Instead,
-  Kong uses the `$realip_remote_addr` variable to append the real IP address
-  of a client, instead of `$remote_addr`, which can come from a previous proxy
-  hop.
-  [#2236](https://github.com/Kong/kong/pull/2236)
-- CNAME records are now properly being cached by the DNS resolver. This results
-  in a performance improvement over previous 0.10 versions.
-  [#2303](https://github.com/Kong/kong/pull/2303)
-- When using Cassandra, some migrations would not be performed on the same
-  coordinator as the one originally chosen. The same migrations would also
-  require a response from other replicas in a cluster, but were not waiting
-  for a schema consensus beforehand, causing indeterministic failures in the
-  migrations, especially if the cluster's inter-nodes communication is slow.
-  [#2326](https://github.com/Kong/kong/pull/2326)
-- The `cassandra_timeout` configuration property is now correctly taken into
-  consideration by Kong.
-  [#2326](https://github.com/Kong/kong/pull/2326)
-- Correctly trigger plugins configured on the anonymous Consumer for anonymous
-  requests (from auth plugins with the new `config.anonymous` parameter).
-  [#2424](https://github.com/Kong/kong/pull/2424)
-- When multiple auth plugins were configured with the recent `config.anonymous`
-  parameter for "OR" authentication, such plugins would override each other's
-  results and response headers, causing false negatives.
-  [#2222](https://github.com/Kong/kong/pull/2222)
-- Ensure
```

Comment:
Maybe instead of removing the titles and this section, we can do:

```
- [2.x](#CHANGELOG-OLD.md)
- [1.x](#CHANGELOG-OLD.md)
- [0.x](#CHANGELOG-OLD.md)
```

This seems to be easier to find and more familiar to existing audiences.

Reply:
![kong-CHANGELOG-md-at-05e9b4575781211c33710973b6e93f6bcc2775cf-·-Kong-kong-·-GitHub](https://user-images.githubusercontent.com/4694585/230663521-8f30c05c-5205-4fcd-82f0-f8c70e27d6b2.png)


^ This is how the changelog page looks today. Is that sufficient?